src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/Extensions.cs

```diff
@@ -165,20 +166,74 @@ public static bool TryGetRegisterMember (this MethodDefinition md, out string me
 			return true;
 		}
 
+		public static TypeDefinition GetMarshalMethodsType (this TypeDefinition type)
+		{
+			foreach (var nt in type.NestedTypes) {
+				if (nt.Name == "__<$>_jni_marshal_methods")
```

Comment:
Do we care about case insensitive comparisions?

Reply:
We want case sensitive comparison here. (we generate that class)